markdown/3.x/en/ff-filter-cloud.md

```diff
@@ -1,14 +1,17 @@
-## Using the Filter Cloud
-You can use the Filter Cloud by adding one line to your HTML code. Doing so will cause the Filter Cloud to use a default HTML template,
-that is `<span data-template="filter">{{element.name}}</span>`, for all filters. Filter Cloud is meant to be used together with ASN,
-so it will work only when `ff-asn` is present on the page.
+## Overview
+Filter Cloud displays all filters that are currently selected through `ff-asn`. As the component is meant to be used together with ASN, it will work only when `ff-asn` is present on the page.
 
 ## Specifying templates
+
 ### Setup
-Leaving the element completely empty makes it possible to omit the otherwise required `data-template="filter"` template. A default template will be applied.
+You can use `ff-filter-cloud` by adding one line to your HTML code:
 ```html
 <ff-filter-cloud></ff-filter-cloud>
 ```
+Leaving the element completely empty makes it possible to omit the otherwise required `data-template="filter"` template. A default template will be applied for all filters:
```

Comment:
_[Nice to have]_
`data-template="filter"` hasn't been introduced yet at this point and might be confusing for someone who reads this article without any prior knowledge about this component. As the possibility of omission is properly explained further below, it can safely be removed here.

Reply:
Please, check now.